@here/harp-mapview/lib/Tile.ts

```diff
@@ -653,6 +656,10 @@ export class Tile implements CachedResource {
         // to be overridden by subclasses
     }
 
+    set skipRender(skipRender: boolean) {
```

Comment:
`willRender` is a callback method whereas `skipRender` is a property. I suggest to rename it to `skipRendering`. Since there is no harm if people read this property you could also just rename `m_skipRendering` and make it public.